src/amo/pages/Guides/index.js

```diff
@@ -312,20 +311,19 @@ export class GuidesBase extends React.Component<InternalProps> {
   getGuidesSections = (
     sections: Array<SectionsType>,
   ): React.ChildrenArray<React.Node> => {
-    const { addons, clientApp, i18n, lang } = this.props;
+    const { addons, i18n } = this.props;
+
+    const splitOn = '__LINK__';
```

Comment:
I don't think this file should have to know this value at all. It could all be internal to the `getLocalizedTextWithLinkParts` function. I'll explain more in other comments.